docs/reference/errors-and-warnings/NU3037.md

```diff
@@ -0,0 +1,23 @@
+---
+title: NuGet Warning NU3037
+description: NU3037 Warning code
+author: heng-liu
+ms.author: henli
+ms.date: 2/21/2019
+ms.topic: reference
+ms.reviewer: dtivel
+f1_keywords: 
+  - NU3037
+---
+
+# NuGet Warning NU3037
+
+
+### Issue
+
+The signature has expired as the certificate used to sign the package has expired.
+
+
+### Solution
+
+If in require mode, only packages signed by a trusted signer will pass validation. So please request the package author to re-sign the package using the `nuget sign` command as described in [NuGet docs](https://docs.microsoft.com/en-us/nuget/create-packages/sign-a-package) using a valid Timestamp Authority signing certificate.Otherwise, just ignore the warning.
```

Comment:
### Issue
A NuGet package signature has expired.
A package signature shares the same validity period as the certificate used to generate the signature.  A package signature is invalid outside of that validity period.
When NuGet’s [signature validation mode] (https://docs.microsoft.com/en-us/nuget/consume-packages/installing-signed-packages#configure-package-signature-requirements) is set to `accept` (default), a package with an expired package signature is treated as an unsigned package and installed anyway.  NU3027 is raised as a warning.
When NuGet’s signature validation mode is set to `require`, a package with an expired package signature is blocked from installation, and NU3027 is elevated from a warning to an error.  Require mode requires non-expired package signatures.
To ensure long-term validity --- even beyond the signing certificate’s validity period --- a package signature should be timestamped with a trusted timestamp.  Trusted timestamps must be added while a package signature is still valid and not expired.

### Solution
* [Resign the package](https://docs.microsoft.com/en-us/nuget/create-packages/sign-a-package) with a non-expired certificate.  Optionally, add a trusted timestamp at the time of signing to ensure long-term validity of the signature.
* For accept mode only, ignore the warning.